crates/ruff_linter/resources/test/fixtures/ruff/RUF035_whitelisted_markup_calls.py

```diff
@@ -0,0 +1,9 @@
+from bleach import clean
+from markupsafe import Markup
+
+content = "<script>alert('Hello, world!')</script>"
+Markup(clean(content))
+
+# indirect assignments are currently not supported
+cleaned = clean(content)
+Markup(cleaned)  # RUF035
```

Comment:
I considered making this work. But then we should probably also make the same thing work with string/bytes literals. At which point I would have to change some of the examples, so they still make sense. So I held off for now.

Reply:
I think it's fine for now to limit this to only direct calls.